src/test/shell/integration/validation_actions_test.sh

```diff
@@ -553,9 +551,36 @@ EOF
 echo "aspect validation output" > $1
 EOF
   chmod +x aspect/aspect_validation_tool
+}
+
+function test_validation_actions_in_rule_and_aspect_no_use_validation_aspect() {
```

Comment:
Mind testing here that both validation actions (in the rule and in the aspect) were executed?

Reply:
Good point, done!